packages/core/src/common/logger-protocol.ts

```diff
@@ -109,6 +109,8 @@ export namespace ConsoleLogger {
     export function log(name: string, logLevel: number, message: string, params: any[]): void {
         const console = consoles.get(logLevel) || originalConsoleLog;
         const severity = (LogLevel.strings.get(logLevel) || 'unknown').toUpperCase();
-        console(`${name} ${severity} ${message}`, ...params);
+        const now = new Date();
+        const timestamp = `${now.toLocaleString(undefined, { hour12: false })}.${now.getMilliseconds()}`;
```

Comment:
Leaving `locale` at undefined will lead to unexpected behavior on different systems. `en-GB` seems to be the most sensible, as it uses `day/month/year` with a 24-hour cycle (no `AM/PM`) by default.

```suggestion
        const timestamp = `${now.toLocaleString('en-GB')}.${now.getMilliseconds()}`;
```